bin/reth/src/lib.rs

```diff
@@ -151,9 +151,11 @@ pub mod rpc {
         pub use reth_rpc_builder::*;
     }
 
-    /// Re-exported from `reth_rpc_types`.
+    /// Re-exported from `alloy_rpc_types`.
     pub mod types {
-        pub use reth_rpc_types::*;
+        /// Alias for a peer identifier
+        pub type PeerId = reth_primitives::B512;
```

Comment:
not sure about this, let me know if you want it somewhere else

Reply:
this is already duplicated as a type in a lot of places, so it's available in a re-export already, not sure we need to duplicate it here

see e.g. https://github.com/search?q=repo%3Aparadigmxyz%2Freth%20peerid&type=code

Reply:
true, just removed it